{"context": " More than 1 year has passed since last update.PHP Error[2]: PDO::__construct(): [2002] No such file or directory (trying to connect via unix:///var/mysql/mysql.sock) ...\n\n\u307f\u305f\u3044\u306a\u30a8\u30e9\u30fc\u304c\u51fa\u305f\u3068\u304d\u306f /protected/config/console.php \u306b\u3066\u3001\u30c7\u30fc\u30bf\u30d9\u30fc\u30b9\u63a5\u7d9a\u306e connectionString \u306e\u30db\u30b9\u30c8\u540d\u3092 localhost \u304b\u3089 127.0.0.1 \u3092\u66f8\u304d\u63db\u3048\u3066\u3084\u308c\u3070\u3044\u3044\u307f\u305f\u3044\u3067\u3059\u3002\u3053\u308c\u306f\u304a\u305d\u3089\u304fYii Framework\u306b\u9650\u3063\u305f\u554f\u984c\u3067\u306f\u306a\u3044\u3068\u601d\u3044\u307e\u3059\u3002\n\n\u53c2\u8003\u30ea\u30f3\u30af\n\nStackoverflow: Error when connecting to MySQL using PHP/PDO\nYii Forum: MySQL connection error generating model\n\n~~~\nPHP Error[2]: PDO::__construct(): [2002] No such file or directory (trying to connect via unix:///var/mysql/mysql.sock) ...\n~~~\n\n\u307f\u305f\u3044\u306a\u30a8\u30e9\u30fc\u304c\u51fa\u305f\u3068\u304d\u306f `/protected/config/console.php` \u306b\u3066\u3001\u30c7\u30fc\u30bf\u30d9\u30fc\u30b9\u63a5\u7d9a\u306e `connectionString` \u306e\u30db\u30b9\u30c8\u540d\u3092 `localhost` \u304b\u3089 `127.0.0.1` \u3092\u66f8\u304d\u63db\u3048\u3066\u3084\u308c\u3070\u3044\u3044\u307f\u305f\u3044\u3067\u3059\u3002\u3053\u308c\u306f\u304a\u305d\u3089\u304fYii Framework\u306b\u9650\u3063\u305f\u554f\u984c\u3067\u306f\u306a\u3044\u3068\u601d\u3044\u307e\u3059\u3002\n\n## \u53c2\u8003\u30ea\u30f3\u30af\n\n* [Stackoverflow: Error when connecting to MySQL using PHP/PDO](http://stackoverflow.com/questions/1819592/error-when-connecting-to-mysql-using-php-pdo)\n* [Yii Forum: MySQL connection error generating model](http://www.yiiframework.com/forum/index.php/topic/8499-mysql-connection-error-generating-model/page__p__42449__hl__pdo+error+con+ole#entry42449)", "tags": ["PHP", "Yii"]}